src/components/shared/DataTableHeader.js

```diff
@@ -10,7 +10,11 @@ const DataTableHeader = ({ headerGroups, onSortedChange, sorted, showFilter }) =
           {headerGroup.headers.map((column) => (
             <Col
               key={column.id}
-              onClick={() => onSortedChange([{ id: column.id, desc: sorted[0]?.id === column.id ? !sorted[0]?.desc : false }])}
+              onClick={() => {
+                if (!dynamicAttributesFromDataTable.includes(column.id)) {
```

Comment:
There is no visual indication of what can be sorted and what can't, right?

Reply:
No, I'll create some sort of "non-intractable" look for the column header!